package.json

```diff
@@ -131,8 +131,8 @@
     "compile-contract-types": "yarn compile-external-abi-types && yarn compile-v3-contract-types",
     "compile-external-abi-types": "typechain --target ethers-v5 --out-dir src/abis/types './src/abis/**/*.json'",
     "compile-v3-contract-types": "typechain --target ethers-v5 --out-dir src/types/v3 './node_modules/@uniswap/?(v3-core|v3-periphery)/artifacts/contracts/**/*.json'",
-    "build": "yarn compile-contract-types && yarn graphql:generate && yarn i18n:extract && yarn i18n:compile && react-scripts build",
-    "i18n:extract": "lingui extract --locale en-US",
+    "build": "yarn compile-contract-types && yarn graphql:generate && yarn i18n:compile && react-scripts build",
+    "i18n:extract": "lingui extract",
     "i18n:compile": "lingui compile",
     "integration-test": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run --record'",
     "graphql:generate": "graphql-codegen --config codegen.yml",
```

Comment:
can remove `touch src/locales/en-US.po` from `prestart` (github won't let comment on a line you didn't modify?). That was a hack for first-time builds when not checking in the `en-US`.